Python/xTelescope.py

```diff
@@ -89,9 +89,9 @@ def OnFirstUpdate(self):
     def Load(self):
         global boolScopeOperated
 
-        solo = true
+        solo = True
         if len(PtGetPlayerList()):
-            solo = false
+            solo = False
```

Comment:
This could be simplified to `solo = bool(PtGetPlayerList())`

Reply:
`solo = not PtGetPlayerList()`, correct?

Reply:
Yes, sorry about that 😢